### PR TITLE
Fix missing fields of types in tests

### PR DESCRIPTION
**Pull request by ferhatelmas**

### CODE REVIEW

File: test/integration/test.js [hunk 1]
```diff
@@ -2272,8 +2272,8 @@ describe('Chat', () => {
 
 		it('Update Message', () => {
 			const c = authClient.channel('twitch', 'state');
-			const message = { id: 1, text: 'my message' };
-			const message2 = { id: 2, text: 'my message 2' };
+			const message = { id: 1, text: 'my message', created_at: new Date() };
+			const message2 = { id: 2, text: 'my message 2', created_at: new Date() };
```
Comment (mahboubii) [inline test/integration/test.js:2276]:
```suggestion
			const message2 = { id: 2, text: 'my message 2', created_at: message.created_at };
```

File: test/integration/test.js [hunk 2]
```diff
@@ -2283,8 +2283,8 @@ describe('Chat', () => {
 
 		it('Add A Message', () => {
 			const c = authClient.channel('twitch', 'state');
-			const message = { id: 1, text: 'my message' };
-			const message2 = { id: 2, text: 'my message 2' };
+			const message = { id: 1, text: 'my message', created_at: new Date() };
+			const message2 = { id: 2, text: 'my message 2', created_at: new Date() };
 			c.state.messages = Immutable([message]);
 			// this should append
 			c.state.addMessageSorted(message2, true);
```
Comment (mahboubii) [inline test/integration/test.js:2290]:
```suggestion
			c.state.addMessageSorted(message2);
```

File: test/integration/test.js (hunk 1 above)
Comment (ferhatelmas) [reply test/integration/test.js]: dropped

File: test/integration/test.js (hunk 2 above)
Comment (ferhatelmas) [reply test/integration/test.js]: dropped

**Review by mahboubii — APPROVED** [review]
🌮

**Pull request merged**